distribution/docker/README.md

```diff
@@ -0,0 +1,3 @@
+## Build
+
+From the root of the repo, run `docker build -t druid:tag -f distribution/docker/Dockerfile .`
```

Comment:
If you go with the cache route this will need to be

```
DOCKER_BUILDKIT=1 docker build -t druid:tag -f distribution/docker/Dockerfile .
```

Reply:
As above, my pref is not to use the docker buildkit since its not universal yet.